src/__tests__/patients/diagnoses/ViewDiagnosis.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { mount } from 'enzyme'
+import { createMemoryHistory } from 'history'
+import React from 'react'
+import { act } from 'react-dom/test-utils'
+import { Route, Router } from 'react-router-dom'
+
+import DiagnosisForm from '../../../patients/diagnoses/DiagnosisForm'
+import ViewDiagnosis from '../../../patients/diagnoses/ViewDiagnosis'
+import PatientRepository from '../../../shared/db/PatientRepository'
+import Patient from '../../../shared/model/Patient'
+
+describe('View Diagnosis', () => {
+  const patient = {
+    id: 'patientId',
+    diagnoses: [{ id: '123', name: 'some name', diagnosisDate: new Date().toISOString() }],
```

Comment:
If we extract the diagnosis to a variable, we can reference that variable everywhere where `patient.diagnoses![0]` exists. We can get rid of the linter warnings and simplify the code.